packages/mux-player/src/errors.ts

```diff
@@ -1,166 +1,182 @@
-import { MediaError } from '@mux/mux-video';
-// @ts-ignore
-import lang from '../lang/en.json';
-import { i18n, parseJwt } from './utils';
+// import lang from '../lang/en.json';
 import type { DialogOptions, DevlogOptions } from './types';
+import { errorCategoryToTokenNameOrPrefix, i18n, MediaError, MuxErrorCategory, MuxErrorCode } from '@mux/playback-core';
 
-export function getErrorLogs(
-  error: MediaError,
-  offline?: boolean,
-  playbackId?: string,
-  playbackToken?: string,
-  translate?: boolean
-): { dialog: DialogOptions; devlog: DevlogOptions } {
-  let dialog: DialogOptions = {};
-  let devlog: DevlogOptions = {};
+const capitalizeFirstLetter = (string: string) => string.charAt(0).toUpperCase() + string.slice(1);
+// NOTE: Since playback-core and mux-video have no devlog or dialog, these translations should still live in Mux Player (at least for now).
 
-  switch (error.code) {
-    case MediaError.MEDIA_ERR_NETWORK: {
-      dialog.title = i18n(`Network Error`, translate);
-      dialog.message = error.message;
+const muxMediaErrorToDialogTitle = (mediaError: MediaError, translate = false) => {
```

Comment:
NOTE: For the error dialog text, I broke each text section (i.e. title + message) into separate functions that translate from code + category into the corresponding text. If this feels a bit too unwieldy, we can refactor to just have a single lookup function for both, though this does mean we'll have redundant messages that we'll need to keep in sync (particularly for titles).